### PR TITLE
TollRideChanges

### PR DESCRIPTION
**Pull request by Aj-cmd-afk**

### CODE REVIEW

**Review by prakharritik — APPROVED** [review]

**Review by prakharritik — APPROVED** [review]

File: src/main/java/base/BaseClass.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ else if(udids.size()==3 && flowOption ==3) {
 			 driverUdid = System.getProperty("driverUdid");
 			 userUdid = System.getProperty("userUdid");
 
-			URL url = new URL("http://0.0.0.0:4723/wd/hub/");
+			URL url = new URL("http://0.0.0.0:4725/wd/hub/");
```
Comment (Gowthami074) [inline src/main/java/base/BaseClass.java:56]: @Aj-cmd-afk No need to push this change

File: movingTech.NY/Driver/DriverLoginFlow.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void successfulDriverLogin() throws InterruptedException {
 		Thread.sleep(2000);
 		System.out.println("Entering Mobile Number");
 
-		driver.findElement(AppiumBy.xpath("//android.widget.EditText[@text='10-digit mobile number']")).sendKeys("9876544445"); 
+		driver.findElement(AppiumBy.xpath("//android.widget.EditText[@text='10-digit mobile number']")).sendKeys("8337860003");
```
Comment (Gowthami074) [inline movingTech.NY/Driver/DriverLoginFlow.java:58]: @Aj-cmd-afk no need to push this change

File: movingTech.NY/Driver/StartRideScreen.java [hunk 3]
```diff
@@ -48,6 +48,11 @@ public void startRide() throws InterruptedException {
         Thread.sleep(4000);
         driver.findElement(AppiumBy.xpath("//android.widget.TextView[@text='Start Ride']")).click();
         Thread.sleep(4000);
+    }
+	
+	public void tollRideStartRideScreen() {
+
```
Comment (Gowthami074) [inline movingTech.NY/Driver/StartRideScreen.java:54]: @Aj-cmd-afk is there  any changes in the StartRideScreen or added extra spaces?

File: movingTech.NY/User/Android/RideSearch.java [hunk 4]
```diff
@@ -11,70 +12,114 @@
...
+        Thread.sleep(2000);
+        driver1.findElement(AppiumBy.xpath("//android.widget.EditText[@text='Where to?']")).sendKeys("Tumu");
+        Thread.sleep(2000);
+        driver1.findElement(AppiumBy.xpath("//android.widget.TextView[@text='Tumukuru']")).click();
+        Thread.sleep(2000);
+        driver1.findElement(AppiumBy.xpath("//android.widget.TextView[@text='Confirm Location']")).click();
+    }
+
+    @Test
+    public void TollRouteRideSearch() throws InterruptedException {
```
Comment (Gowthami074) [inline movingTech.NY/User/Android/RideSearch.java:92]: @Aj-cmd-afk Why are there two TollRouteRideSearch() methods in the RideSearch class?

File: movingTech.NY/Driver/StartRideScreen.java (hunk 3 above)
Comment (Aj-cmd-afk) [reply movingTech.NY/Driver/StartRideScreen.java:54]: It is just extra Space

**Review by Gowthami074 — APPROVED** [review]

File: movingTech.NY/User/Android/RideSearch.java (hunk 4 above)
Comment (Aj-cmd-afk) [reply movingTech.NY/User/Android/RideSearch.java:92]: its a typo error, will correct in next PR along with other changes.

**Pull request merged**